modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -23,6 +23,8 @@ GA:: General Availability
 
 | Operator | Pipelines | Triggers | CLI | Chains | Hub | {pac} | Results | Manual Approval Gate | |
 
+|1.18 | 0.68.x | 0.31.x | 0.40.x | 0.23.x (GA) | 1.20.x (TP) | 0.33.x (GA) | 0.14.x (GA) | 0.5.x (TP) | 4.15, 4.16, 4.17 4.18 | GA
+
 |1.17 | 0.65.x | 0.30.x | 0.39.x | 0.23.x (GA) | 1.19.x (TP) | 0.29.x (GA) | 0.13.x (TP) | 0.4.x (TP) | 4.15, 4.16, 4.17 | GA
 
 |1.16 | 0.62.x | 0.29.x | 0.38.x | 0.22.x (GA) | 1.18.x (TP) | 0.28.x (GA) | 0.12.x (TP) | 0.3.x (TP) | 4.15, 4.16, 4.17 | GA
```

Comment:
Maybe we could add OpenShift 4.18 to 1.16 and 1.17. (It wasn't GA when we were working on those releases)